### DISCUSSION TITLE
Custom colorscheme for gruvbox

### DISCUSSION THREAD

**Discussion by dR3b** [nix-community/nixvim#990] [General]
Hi, 
Is it possible to change the gruvbox colors?

https://github.com/ellisonleao/gruvbox.nvim/blob/6e4027ae957cddf7b193adfaec4a8f9e03b4555f/lua/gruvbox.lua#L65

Example:
```
palette_overrides = {
  dark1 = "#323232",
  dark2 = "#383330",
  dark3 = "#323232",
  bright_blue = "#5476b2",
  bright_purple = "#fb4934",
}
```
Thanks!

**Comment by GaetanLepage** [comment]
This is now possible thanks to #991.
You can set `plugins.gruvbox.settings.palette_overrides = {<WHATEVER>};`

**Comment by dR3b** [comment]
Thanks! I will try it!
